pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -1,30 +1,32 @@
-{ dpkg, fetchurl, lib, stdenv }:
+{ squashfsTools, fetchurl, lib, stdenv }:
 
+# This derivation roughly follows the update-ffmpeg script that ships with the official Vivaldi
+# downloads at https://vivaldi.com/download/
 stdenv.mkDerivation rec {
   pname = "chromium-codecs-ffmpeg-extra";
-  version = "112.0.5615.49";
+  version = "111306";
 
   src = fetchurl {
-    url = "https://launchpadlibrarian.net/660647727/${pname}_${version}-0ubuntu0.18.04.1_amd64.deb";
-    sha256 = "sha256-mHjvjRG+toRcsOMca+JPXNZPgyQROH2qtSpBPHLmt3s=";
+    url = "https://api.snapcraft.io/api/v1/snaps/download/XXzVIXswXKHqlUATPqGCj2w2l7BxosS8_34.snap";
```

Comment:
Do we really need to use snap?

Reply:
I'm not happy with it, but for now I'm afraid so. It's what the Vivaldi official download uses for both its deb and rpm file. In [this forum thread](https://forum.vivaldi.net/topic/88884/page-crash-due-to-read-out-of-range-on-nixos) I've been asking around for a better way. I intend to improve it once  I dive deeper into it. For now, however, I would like to get a fix merged quickly, to get a working vivaldi again.